README.md

```diff
@@ -22,13 +22,34 @@ Built with the Next.js 14 App Router, tRPC, TypeScript, Payload & Tailwind
 
 ## Getting started
 
-To get started with this project, run
+### 1. Fork and Clone the Repository
 
 ```bash
   git clone https://github.com/joschan21/digitalhippo.git
```

Comment:
Or alternativly you can add ` . ` dir at last of cmd!


Linked with :  https://github.com/joschan21/digitalhippo/pull/8#discussion_r1407932982

Reply:
cd ./digitalhippo is better as its easy to understand